OpenEdXMobile/res/layout/layout_course_dashboard_certificate.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/neutralWhite"
+    android:gravity="center"
+    android:orientation="vertical"
+    android:padding="@dimen/edx_default_margin">
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:layout_width="@dimen/course_certificate_logo_width"
```

Comment:
As per designs, the width is 138dp but we can make it "match_parent" and only the height needs to be set to "100dp" as we are using a vector image and it will map it accordingly. Right now it is showing extra space on the right side of the image.

Reply:
Dimens were updated to 138dp instead.